rasa/telemetry.py

```diff
@@ -563,6 +563,27 @@ def toggle_telemetry_reporting(is_enabled: bool) -> None:
 
     rasa_utils.write_global_config_value(CONFIG_FILE_TELEMETRY_KEY, configuration)
 
+def filter_errors(event: Dict[Text, Any], _unused_hint: Optional[Dict[Text, Any]] = None) -> None:
+    """Filtering errors.
+
+    Args:
+        event: event to be logged to sentry
+        _unused_hint: some hinting information sent alongside of the event
+
+    Returns:
+        the event without any sensitive / PII data or `None` if the event should
+        be discarded.
+    """
+    if 'exc_info' in _unused_hint:
+        exc_type, exc_value, tb = _unused_hint['exc_info']
+        if isinstance(exc_value, ImportError):
+            return None
+    return event
+
+def before_send(event: Dict[Text, Any], _unused_hint: Optional[Dict[Text, Any]] = None) -> Optional[Dict[Text, Any]]:
+  event = filter_errors(event, _unused_hint)
+  event = strip_sensitive_data_from_sentry_event(event, _unused_hint)
```

Comment:
As discussed in the thread, it's best to re-order these function calls, so that `strip_sensitive_data_from_sentry_event` runs first, then `filter_errors`.

Reply:
How did I miss this? Valid point though. Nice catch 🚀